src/NServiceBus.NHibernate/TimeoutPersisters/NHibernateTimeoutStorage.cs

```diff
@@ -54,10 +56,18 @@ protected override void Setup(FeatureConfigurationContext context)
                 }
             }
 
+            var timeoutsCleanupExecutionInterval = 
+                context.Settings.GetOrDefault<TimeSpan?>("NHibernate.Timeouts.CleanupExecutionInterval") ?? TimeSpan.FromMinutes(2);
+
+            var timeoutsCleanupQueryPeriod =
+                context.Settings.GetOrDefault<TimeSpan?>("NHibernate.Timeouts.CleanupQueryPeriod") ?? TimeSpan.FromMinutes(5);
```

Comment:
Why do we need this? When the endpoint starts we query from 10 years before (AFAIR) so I guess here we can do the same thing?